Velaptor/OpenGL/RectBatchItem.cs

```diff
@@ -0,0 +1,212 @@
+    /// </para>
+    /// </remarks>
+    public RectBatchItem(
+        Vector2 position = default,
+        float width = 1f,
+        float height = 1f,
+        Color color = default,
+        bool isFilled = true,
+        float borderThickness = 1f,
+        CornerRadius cornerRadius = default,
+        ColorGradient gradientType = ColorGradient.None,
+        Color gradientStart = default,
+        Color gradientStop = default,
+        int layer = 0)
+    {
+        Position = position;
+        Width = width;
+        Height = height;
+        Color = color == Color.Empty ? Color.White : color;
+        IsFilled = isFilled;
+        BorderThickness = borderThickness;
+
+        if (cornerRadius == CornerRadius.Empty())
+        {
+            CornerRadius = new CornerRadius(1, 1, 1, 1);
+        }
+        else
+        {
+            var minValue = Width > height ? width : height;
+
+            cornerRadius = cornerRadius.TopLeft > minValue
+                ? CornerRadius.SetTopLeft(cornerRadius, minValue)
+                : cornerRadius;
+
+            cornerRadius = cornerRadius.TopRight > minValue
+                ? CornerRadius.SetTopRight(cornerRadius, minValue)
+                : cornerRadius;
+
+            cornerRadius = cornerRadius.BottomRight > minValue
+                ? CornerRadius.SetBottomRight(cornerRadius, minValue)
+                : cornerRadius;
+
+            cornerRadius = cornerRadius.BottomLeft > minValue
+                ? CornerRadius.SetBottomLeft(cornerRadius, minValue)
+                : cornerRadius;
+
+            CornerRadius = cornerRadius;
+        }
+
+        GradientType = gradientType;
+        GradientStart = gradientStart == Color.Empty ? Color.White : gradientStart;
+        GradientStop = gradientStop == Color.Empty ? Color.White : gradientStop;
+        Layer = layer;
+    }
+
+    /// <summary>
+    /// Gets the position of the rectangle.
+    /// </summary>
+    /// <remarks>
+    ///     This is the center of the rectangle.
+    /// </remarks>
+    public Vector2 Position { get; }
+
+    /// <summary>
+    /// Gets the width of the rectangle.
+    /// </summary>
+    /// <remarks>
+    ///     The width is restricted to a minimum value of 1.
+    /// </remarks>
+    public float Width { get; }
+
+    /// <summary>
+    /// Gets the height of the rectangle.
+    /// </summary>
+    /// <remarks>
+    ///     The height is restricted to a minimum value of 1.
+    /// </remarks>
+    public float Height { get; }
+
+    /// <summary>
+    /// Gets the color of the rectangle.
+    /// </summary>
+    /// <remarks>
+    ///     Ignored if the <see cref="GradientType"/> is set to any value other than <see cref="ColorGradient.None"/>.
+    /// </remarks>
+    public Color Color { get; }
+
+    /// <summary>
+    /// Gets a value indicating whether or not the rectangle is filled or empty.
```

Comment:
Refactor the code docs

```suggestion
    /// Gets a value indicating whether or not the rectangle is filled with a solid color.
```